cloudant-client/src/main/java/com/cloudant/client/api/Changes.java

```diff
@@ -115,54 +148,104 @@ public void stop() {
      * @return {@link ChangesResult} encapsulating the normal feed changes
      */
     public ChangesResult getChanges() {
-        com.cloudant.client.org.lightcouch.ChangesResult couchDbChangesResult = changes
-                .getChanges();
-        ChangesResult changeResult = new ChangesResult(couchDbChangesResult);
-        return changeResult;
+        final URI uri = this.databaseHelper.changesUri("feed", "normal");
+        return client.get(uri, ChangesResult.class);
     }
 
     // Query Params

```

Comment:
That was just a direct code move of the duplicate lightcouch methods into a single class, neither of which was documented. I agree they should be documented though so I have added docs in 088dbcc.